test/cypress/integration/Highcharts/annotations/fib-time-zones/timezones.spec.js

```diff
@@ -0,0 +1,110 @@
+describe('Stock Tools Fibonacci Tome Zones, #15825', () => {
+    beforeEach(() => {
+        cy.viewport(1000, 500);
```

Comment:
After fixing the issue with flicking annotation on the ordinal axis please add a test.